<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsToolBoxUi</class>
 <widget class="QWidget" name="xToolsToolBoxUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QWidget" name="widgetControl" native="true">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDockWidget" name="dockWidget_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="features">
         <set>QDockWidget::NoDockWidgetFeatures</set>
        </property>
        <property name="allowedAreas">
         <set>Qt::NoDockWidgetArea</set>
        </property>
        <property name="windowTitle">
         <string>Communication control</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents_3">
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widgetCommunicationToolUi" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButtonCommunicationSettings">
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCommunicationOpen">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QDockWidget" name="dockWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="features">
         <set>QDockWidget::NoDockWidgetFeatures</set>
        </property>
        <property name="allowedAreas">
         <set>Qt::NoDockWidgetArea</set>
        </property>
        <property name="windowTitle">
         <string>Output control</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents">
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="xToolsTextFormatComboBox" name="comboBoxOutputFormat"/>
          </item>
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="1">
             <widget class="xToolsCheckBox" name="checkBoxOutputTx">
              <property name="text">
               <string>Tx</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="xToolsCheckBox" name="checkBoxOutputMs">
              <property name="text">
               <string>MS</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="xToolsCheckBox" name="checkBoxOutputWrap">
              <property name="text">
               <string>Wrap</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="xToolsCheckBox" name="checkBoxOutputRx">
              <property name="text">
               <string>Rx</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="xToolsCheckBox" name="checkBoxOutputDate">
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="xToolsCheckBox" name="checkBoxOutputTime">
              <property name="text">
               <string>Time</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="xToolsCheckBox" name="checkBoxFromTo">
              <property name="toolTip">
               <string>Show the data origin or sending target</string>
              </property>
              <property name="text">
               <string>From-To</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="xToolsCheckBox" name="checkBoxConsoleMode">
              <property name="toolTip">
               <string>Console mode</string>
              </property>
              <property name="text">
               <string>Console</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButtonOutputSettings">
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonOutputClear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDockWidget" name="dockWidget_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="features">
         <set>QDockWidget::NoDockWidgetFeatures</set>
        </property>
        <property name="allowedAreas">
         <set>Qt::NoDockWidgetArea</set>
        </property>
        <property name="windowTitle">
         <string>Input control</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents_2">
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Cycle</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxInputIntervel">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="xToolsTextFormatComboBox" name="comboBoxInputFormat"/>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButtonInputSend">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButtonInputSettings">
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabIO">
      <attribute name="title">
       <string>I/O</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="6">
        <widget class="QTextBrowser" name="textBrowserOutput"/>
       </item>
       <item row="3" column="5">
        <widget class="QToolButton" name="toolButtonPrestore">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: &quot;blue&quot;
}</string>
           </property>
           <property name="text">
            <string>Rx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="xToolsStatisticianToolUi" name="widgetRxStatistician" native="true">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="xToolsVelometerToolUi" name="widgetRxVelometer" native="true">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: &quot;green&quot;
}</string>
           </property>
           <property name="text">
            <string>Tx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="xToolsStatisticianToolUi" name="widgetTxStatistician" native="true">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="xToolsVelometerToolUi" name="widgetTxVelometer" native="true">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="6">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Input</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelAlgorithm">
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string notr="true">:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelCrc">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="xToolsComboBox" name="comboBoxInputText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEmiter">
      <attribute name="title">
       <string>Emiter</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabResponser">
      <attribute name="title">
       <string>Responser</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPrestorer">
      <attribute name="title">
       <string>Prestorer</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabTransmitter">
      <attribute name="title">
       <string>Transmitter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidgetTransmitter">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsTextFormatComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsTextFormatComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">xToolsCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsVelometerToolUi</class>
   <extends>QWidget</extends>
   <header location="global">xToolsVelometerToolUi.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>xToolsStatisticianToolUi</class>
   <extends>QWidget</extends>
   <header location="global">xToolsStatisticianToolUi.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOutputClear</sender>
   <signal>clicked()</signal>
   <receiver>textBrowserOutput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
